doc/source/examples_source/00-basic/07-volume_render.py

```diff
@@ -0,0 +1,199 @@
+session.load_data(
+    data_file=xyz_file,
+    result_file=q_file,
+    file_format="PLOT3D",
+    representation="3D_feature_2D_full",
+)
+session.show("image", width=800, height=600)
+
+###############################################################################
+# The PLOT3D reader only reads the volume by default. Now, extract a
+#  particular IJK range for the surface of the shuttle
+# ------------------------------------------------------------------
+
+sesse.data_partbuild.begin()
+sesse.case.select("Case 1")
+sesse.data_partbuild.data_type("structured")
+sesse.data_partbuild.group("OFF")
+sesse.data_partbuild.select_begin(1)
+sesse.data_partbuild.domain("all")
+sesse.data_partbuild.noderange_i(1, 53)
+sesse.data_partbuild.noderange_j(1, 63)
+sesse.data_partbuild.noderange_k(1, 1)
+sesse.data_partbuild.nodestep(1, 1, 1)
+sesse.data_partbuild.nodedelta(0, 0, 0)
+sesse.data_partbuild.description("Shuttle")
+sesse.data_partbuild.create()
+sesse.part.select_byname_begin("(CASE:Case 1)Shuttle")
+sesse.case.select("Case 1")
+sesse.data_partbuild.end()
+session.show("image", width=800, height=600)
+
+##############################################################################
+# Volume Render Flow
+# ------------------------------------------------------------------
+# 
+# .. image:: /_static/07_volume_render_1.png
+#
+#
+# Make the shuttle surface an uninteresting, uniform color grey
+#
+sesse.part.select_begin(2)
+sesse.part.colorby_palette("none")
+sesse.part.colorby_rgb(0.6,0.6,0.6)
+sesse.view.highlight_parts("OFF")
+#
+# Color the 3D flow field by the variable that we
+#  wish to visualize using Volume Rendering, Mach
+#
+sesse.part.select_begin(1)
+sesse.part.modify_begin()
+sesse.part.colorby_palette("Mach")
+sesse.legend.select_palette_begin("Mach")
+sesse.part.modify_end()
+sesse.legend.select_palette_begin("Mach")
+sesse.function.palette("Mach")
+sesse.function.palette("Mach")
+sesse.function.modify_begin()
+sesse.function.restore_predefinedpal("use_new_levels","Magma")
+sesse.function.modify_end()
+#
+#  Volume Rendering, in EnSight, is accomplished by subdividing
+#   the flow field into a finite number of hex elements and then
+#   doing the rendering of the variable in each box according to
+#   the normalized variable value and the opacity ('alpha') for
+#   ranges of values set using function points and the magnitude
+#
+sesse.tools.box("ON")
+sesse.tools.box("OFF")
+sesse.part.select_begin(1)
+sesse.clip.begin()
+sesse.part.colorby_palette("Mach")
+sesse.clip.domain("volume")
+sesse.clip.tool("xyz_box")
+sesse.clip.sample_step(128,128,128)
+sesse.clip.origin(-0.497170001,1.27204275,-0.879772604)
+#
+sesse.clip.axis("x",0,0,1)
+sesse.clip.axis("y",0,-1,0)
+sesse.clip.axis("z",1,0,0)
+#
+sesse.clip.length(2.24244738,1.39883041,3.60410643)
+sesse.clip.end()
+sesse.clip.create()
+sesse.legend.width(0.0244169608)
+sesse.legend.height(0.662014842)
+sesse.function.palette("Mach")
```

Comment:
I think my overall comment is to focus on trying to show someone how to generate a volume rendering of a variable. Don't worry about legend widths and heights, or weeds along the way. Try to make this as compact and suscint to volume render creation. These code bits are being copy/pasted into workflows, and if there lots of extra stuff here it becomes confusing what you need/don't need for the job at hand.

Reply:
deleted 130-131

Reply:
When I deleted the lines, I got the following image, because it is using a palette with a lot of values displayed. So, I put it back in.
![image](https://github.com/ansys/pyensight/assets/111907585/63709ce8-6919-49a1-9798-f3b1314d7cb2)